scripts-available/CDB_Helper.sql

```diff
@@ -0,0 +1,163 @@
+-- UTF8 safe and lenght aware. Find a unique identifier with a given prefix
+-- and/or suffix and withing a schema.
+CREATE OR REPLACE FUNCTION cartodb._CDB_Unique_Identifier(prefix TEXT, relname TEXT, suffix TEXT, schema TEXT DEFAULT NULL)
```

Comment:
It is customary to add an explanation of the arguments' semantic. Important in some cases, for instance: meaning of the default `schema DEFAULT NULL`.